README.md

```diff
@@ -57,17 +59,37 @@ content-length: 98
 content-type: application/json
 
 {
-    "name": "ut dolore nulla non ex",
```

Comment:
The example specifies the `petstore.json` but this response is not coming from that file. I would make the things consistent so that the flow is easier to follow

Reply:
It is exactly the example.

`./packages/cli/bin/run mock examples/petstore.json`
`(karol@MacBook-Pro-Karol ~)$ curl http://127.0.0.1:4010/pet/123
{"name":"doggie","photoUrls":["dolor laborum","amet eiusmod ut in","comm"],"id":-53524329,"category":{"id":-6563821,"name":"consequat Ut ea"},"tags":[{"id":90196542,"name":"Lorem laborum ullamco"},{"id":-21294836,"name":"eu"}],"status":"sold"}(karol@MacBook-Pro-Karol ~)$`

Reply:
The example generation changed recently so not only required fields are returned.

Reply:
Whoa, is it really the official example? That is so sad…how about replacing it with the oas3 one then? It's way better than the oas2 one

Reply:
Updated with oas3 spec, the example is prettier now :)

Reply:
Fantastic! This is resolved.

Reply:
I'm unresolving conversation :)

The example I used was incomplete. It worked with http://127.0.0.1:4010/pet/123 but the rest of examples in README.md would stop working (missing other response codes, missing operations).

Now I'm quite stuck. I could change the example - but I need to rewrite all usage examples in README. For me it's a candidate for separate task. Moreover I updated the example just to match current state. It's not related to this task.

Reply:
Continue here: https://github.com/stoplightio/prism/pull/244

Reply:
If you ever want more examples, theres a bunch over here https://github.com/Mermade/openapi3-examples